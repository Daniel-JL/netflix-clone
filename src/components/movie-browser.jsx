import React from 'react';

export function MovieBrowser() {

  return (
    <div>
      
    </div>
  )
}

