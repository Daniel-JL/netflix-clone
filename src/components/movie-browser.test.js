import React from 'react';
import { fireEvent, render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { MovieBrowser } from './movie-browser.jsx'

describe('Movie browser component', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<MovieBrowser />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});