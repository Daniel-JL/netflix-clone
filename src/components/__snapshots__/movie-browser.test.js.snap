// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie browser component renders correctly 1`] = `<div />`;
